instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpClientAttributesExtractor.java

```diff
@@ -32,6 +35,9 @@ public final class HttpClientAttributesExtractor<REQUEST, RESPONSE>
         REQUEST, RESPONSE, HttpClientAttributesGetter<REQUEST, RESPONSE>>
     implements SpanKeyProvider {
 
+  private static final Set<String> PARAMS_TO_REDACT =
+      new HashSet<>(Arrays.asList("AWSAccessKeyId", "Signature", "sig", "X-Goog-Signature"));
```

Comment:
Instead of using a literal on line 159, we could define this here too
```suggestion
  private static final String REDACTED_CREDENTIALS = "REDACTED:REDACTED";
  private static final Set<String> PARAMS_TO_REDACT =
      new HashSet<>(asList("AWSAccessKeyId", "Signature", "sig", "X-Goog-Signature"));
```

Reply:
The credential redaction exists before this PR. I would prefer to limit the changes to the scope of this PR.